app/screens/login_options/login_options.js

```diff
@@ -188,6 +188,41 @@ class LoginOptions extends PureComponent {
         return null;
     };
 
+    renderO365Option = () => {
+        const {config, license} = this.props;
+        const forceHideFromLocal = LocalConfig.HideO365LoginExperimental;
+        const o365Enabled = config.EnableSignUpWithOffice365 === 'true' && license.IsLicensed === 'true' && license.Office365OAuth === 'true';
+
+        if (!forceHideFromLocal && o365Enabled) {
+            const backgroundColor = config.EmailLoginButtonColor || '#2389d7';
+            const additionalStyle = {
+                backgroundColor,
+            };
+
+            if (config.EmailLoginButtonBorderColor) {
+                additionalStyle.borderColor = config.EmailLoginButtonBorderColor;
+            }
+
+            const textColor = config.EmailLoginButtonTextColor || 'white';
+
+            return (
+                <Button
+                    key='o365'
```

Comment:
Is this key needed?

Reply:
it doesn't hurt to have it, every other login option already has it